open-banking-accelerator/components/consent-management/com.wso2.openbanking.accelerator.consent.extensions/src/main/java/com/wso2/openbanking/accelerator/consent/extensions/authorize/utils/ConsentRetrievalUtil.java

```diff
@@ -58,7 +58,7 @@ public static String extractRequestObject(String spQueryParams) {
             for (String param : spQueries) {
                 if (param.contains("request=")) {
                     requestObject = (param.substring("request=".length())).replaceAll(
-                            "\\r\\n|\\r|\\n|\\%20", "");
+                            "\\r\\n|\\r|\\n|%20", "");
```

Comment:
what's the reason for this change?

Reply:
This change was made in order to remove a warning notified in the intellij idea as

>  Redundant Character escape '\\%' in RegExp

Reply:
Fixed in [01f2f08]